docs/docsite/rst/rules/default_rules.rst

```diff
@@ -20,7 +20,7 @@ ID
-`E202 <https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/OctalPermissionsRule.py>`_                                                                                                                                                                                                  historic                                                                                                                                                                                                                                                                                                     Octal file permissions must contain leading zero                                                                                                                                                                                                                                                             Numeric file permissions without leading zero can behave in unexpected ways. See http://docs.ansible.com/ansible/file_module.html                                                                                                                                                                            
+`E202 <https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/OctalPermissionsRule.py>`_                                                                                                                                                                                                  historic                                                                                                                                                                                                                                                                                                     Octal file permissions must contain leading zero or be a string                                                                                                                                                                                                                                              Numeric file permissions without leading zero can behave in unexpected ways. See http://docs.ansible.com/ansible/file_module.html                                                                                                                                                                            
```

Comment:
Why does this show up in diff?

Reply:
Running lint seems to reformat the file, it happened twice locally. I had no intention to touch it, still...

Reply:
Plz drop this change and let's deal with it separately. I want to understand what the problem is instead of blindly ignoring it.